.github/workflows/release.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: '>=1.18'
```

Comment:
for the official builds, we'd like to stick with the latest stable version of Go.